extensions/objects/models/objects.py

```diff
@@ -827,3 +827,49 @@ class Fraction(BaseObject):
             'schema': PositiveInt.SCHEMA
         }]
     }
+
+
+class Units(BaseObject):
```

Comment:
Add a comment explaining why validation in backend is not possible as mentioned by [Sean](https://docs.google.com/document/d/13TrooKtS85APB8fKWLfRJtDmTvjyf07E-LeayFn9TMw/edit?disco=AAAABw3MKfs)

Reply:
Done.

Reply:
There is still no comment.

Reply:
It's there in the doc.